torch/ao/nn/quantized/modules/functional_modules.py

```diff
@@ -220,6 +226,12 @@ def add_relu(self, x: Tensor, y: Tensor) -> Tensor:
         r = self.activation_post_process(r)
         return r
 
+    r"""Operation equivalent to ``torch.ops.quantized.matmul(Tensor, Tensor)``"""
+    def matmul(self, x: Tensor, y: Tensor) -> Tensor:
+        r = ops.quantized.matmul(x, y, scale=self.scale, zero_point=self.zero_point)
+        r = self.activation_post_process(r)
```

Comment:
I feel we probably don't need to call this, could you remove this line and test again to see if it still works?

Reply:
Can you elaborate why we don't need `matmul` in QFunctional? the eager mode convert will swap FloatFunctional to QFunctional right?

Reply:
no I mean we don't need to call `        r = self.activation_post_process(r)`